### ISSUE TITLE
KeepInventory state is being reset upon server reboot.

### ISSUE THREAD

**Issue by Neulooo** [rosecat-dev/IndividualKeepInv#3]
This issue is not impacting people that joined after the mod has been updated to 1.0.1 as heavily, as their keepInventory state is being reset to True (server's default), but for other people it's getting set to false, which has already heavily impacted some individuals on the server.

For context, it has been tested on both pre-1.0.1 player who had their keepInventory state set to True and after server restart it went back to False, and on a post-1.0.1 player who had their keepInventory state set to False, which after restart got set to True. I'm not actually sure if this update has affected this behaviour or if it could be noted even prior to that, but still.

**Comment by rosecat-dev** [comment]
Extremely sorry to hear this is happening and that it has affected your players.

I'm honestly not entirely sure what could be causing this at the moment, but I did notice a small error with how the default was being stored, which *might* be it?

[individualkeepinv-1.0.3-TEST.zip](https://github.com/rosecat-dev/IndividualKeepInv/files/12194733/individualkeepinv-1.0.3-TEST.zip)

Try this and see if it helps at all, would be greatly appreciated.

**Comment by rosecat-dev** [comment]
Extremely sorry to hear that this has affected your players. Working on a fix now.

**Comment by rosecat-dev** [comment]
Think I've found the issue. The ``writeNbt`` method responsible for writing player states to disc when a server closes does not appear to be running after the first time the server closes.

**Comment by rosecat-dev** [comment]
[individualkeepinv-1.0.3-TEST.zip](https://github.com/rosecat-dev/IndividualKeepInv/files/12195784/individualkeepinv-1.0.3-TEST.zip)

This should fix it. `writeNbt` runs every time a server closes now. Please tell me if so/if not!

**Comment by Neulooo** [comment]
Over a day has went by and I have come back with results.
Since to install the mod we've had to restart the server, thus resetting everyone's state, it was a bit bothersome having to notify everyone on login and in tab menu that they should change their keepInventory state, but outside of that we've had no reported issues regarding lost items or similar problems.

We do have now a contingency in form of a inventory backup mod, so future issues shouldn't be as impactful.

Also, not sure when this was implemented, but thanks for making the "getdefault" subcommand.
